docs/reference/experiment-config.txt

```diff
@@ -659,8 +659,8 @@ large-scale experiments with hundreds or thousands of trials.
 Adaptive (Simple)
 =================
 
-Note: Adaptive (Simple) will be deprecated. We recommend using
-the state-of-the-art Adaptive (ASHA) searcher.
+Note: Adaptive (Simple) will be deprecated. We recommend using the
```

Comment:
question: Are there any treatments we can use to make this more prominent? Something like the screenshot below, with a more warning-y color?

![Screen Shot 2020-10-26 at 15 51 52](https://user-images.githubusercontent.com/56557451/97236870-433b4d00-17a3-11eb-85d5-c43f3663abae.png)

Reply:
Aha, literally answered my own question by scrolling down a tiny bit :) Can we use this instead, with a link to the topic guide for ASHA?

![Screen Shot 2020-10-26 at 15 52 59](https://user-images.githubusercontent.com/56557451/97236924-61a14880-17a3-11eb-9018-bae3f23c8b47.png)

Reply:
Ooh yeah let me add this